tests/spread/core22/package-repo-archs/armhf.yaml

```diff
@@ -0,0 +1,38 @@
+name: test-archs
```

Comment:
@sergiusens @cmatsuoka this is still a draft-in-progress but I would like to get your opinions on these spread tests: do they reflect what we want to be able to do with package-repositories?